contracts/tests/tests/votes/test_withdraw.rs

```diff
@@ -114,17 +114,17 @@ fn test_withdraw_to_full_balance() {
     let governor = Address::generate(&e);
 
     let (token_id, token_client) = create_stellar_token(&e, &bombadil);
-    let (votes_id, votes_client) = create_wrapped_token_votes(&e, &token_id, &governor);
+    let (votes_id, votes_client) = create_staking_token_votes(&e, &token_id, &governor);
```

Comment:
Should make one of these the soroban token votes to ensure that we are testing the soroban version